auth/src/main/java/com/firebase/ui/auth/util/data/ProviderAvailability.java

```diff
@@ -8,8 +8,6 @@
             exists("com.firebase.ui.auth.data.remote.GitHubSignInHandler");
     public static final boolean IS_FACEBOOK_AVAILABLE =
             exists("com.facebook.login.LoginManager");
-    public static final boolean IS_TWITTER_AVAILABLE =
```

Comment:
So could we still try and detect this and then log a warning saying "by the way you don't need twitter kit anymore"?  Maybe we should also do the same for the `R.string.twitter_` config values we used to have devs put in.  Then we can maybe get more people to actually benefit from this change.

Reply:
Done (TwitterKit check only).